webpack/assets/javascripts/react_app/components/ForemanModal/subcomponents/SubmitOrCancel/SubmitBtn.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Button } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../../../common/I18n';
+import { simpleLoader } from '../../../common/Loader';
+
+const SubmitBtn = ({ submitting, onSubmit }) => (
```

Comment:
From a UI perspective, people don't read the text of dialog boxes; they just read the text of the button and decide to click based on that.  So, the button label should be the action the user is about to do, such as "Delete", etc.  I think that hard-coding it to "Submit" is too all-purpose.  Can I pass the button text as a prop instead of having it set in stone as 'Submit' ?  Also, for destructive things like deleting, I would want to be able to set `bsStyle` to `warning` or `danger`.  Should pass that down too.

(I do really like the use of `simpleLoader` btw)